lib/rides/widgets/search_ride_filter.dart

```diff
@@ -370,16 +365,6 @@ class SearchSuggestionFilter {
           Row(children: _selectedFeatures.map((Feature feature) => feature.getIcon(context)).toList());
       widgets.add(featuresRow);
     }
-    if (!isDeviationDefault) {
-      final Widget deviationWidget = Row(
-        children: <Widget>[
-          const Icon(Icons.schedule),
-          const SizedBox(width: 6),
-          Text('± ${_maxDeviationController.text}')
-        ],
-      );
-      widgets.add(deviationWidget);
-    }
     final Widget sortingWidget = Row(
```

Comment:
I know we wanted to make it so this is not part of the Inkwell, but instead leads to a different dropdown. Do we want to do that in a different PR after the user studies, or now?

Reply:
I'd do it after the user studies